src/screens/ProposalDetail.js

```diff
@@ -295,18 +296,40 @@ const Amount = ({
   requestedAmount = 0,
   requestToken: { symbol, decimals, verified },
 }) => {
+  const signalingProposal = requestedAmount.eq(0)
   const tokenIcon = getTokenIconBySymbol(symbol)
   return (
     <DataField
-      label="Request Amount"
+      label={!signalingProposal && 'Request Amount'}
       value={
-        <Balance
-          amount={requestedAmount}
-          decimals={decimals}
-          symbol={symbol}
-          verified={verified}
-          icon={tokenIcon}
-        />
+        signalingProposal ? (
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+            `}
+          >
+            <img src={signalingBadge} alt="" />
```

Comment:
Let's set the `height` and `width`

```suggestion
            <img src={signalingBadge} height="24" width="24"  alt="" />
```